cmd/manifest/manifest.go

```diff
@@ -10,39 +10,41 @@ import (
 
 type SkupperImage struct {
 	Name       string `yaml:"name"`
+	SHA        string `yaml:"sha"`
 	Repository string `yaml:"repository,omitempty"`
 }
 
 func generateManifestFile() error {
-
-	// Define an array of images.
-	skupperImages := []SkupperImage{
-		{
-			Name:       images.GetRouterImageName(),
-			Repository: "https://github.com/skupperproject/skupper-router",
-		},
-		{
-			Name:       images.GetServiceControllerImageName(),
-			Repository: "https://github.com/skupperproject/skupper",
-		},
-		{
-			Name:       images.GetConfigSyncImageName(),
-			Repository: "https://github.com/skupperproject/skupper",
-		},
-		{
-			Name:       images.GetFlowCollectorImageName(),
-			Repository: "https://github.com/skupperproject/skupper",
-		},
-		{
-			Name: images.GetPrometheusServerImageName(),
-		},
-	}
-
 	// Define a struct for the manifest file.
 	manifest := struct {
 		Images []SkupperImage `json:"images"`
 	}{
-		Images: skupperImages,
+		Images: []SkupperImage{
+			{
+				Name:       images.GetRouterImageName(),
+				SHA:        images.GetSha(images.GetRouterImageName()),
+				Repository: "https://github.com/skupperproject/skupper-router",
+			},
+			{
+				Name:       images.GetServiceControllerImageName(),
+				SHA:        images.GetSha(images.GetServiceControllerImageName()),
+				Repository: "https://github.com/skupperproject/skupper",
+			},
+			{
+				Name:       images.GetConfigSyncImageName(),
+				SHA:        images.GetSha(images.GetConfigSyncImageName()),
+				Repository: "https://github.com/skupperproject/skupper",
+			},
+			{
+				Name:       images.GetFlowCollectorImageName(),
+				SHA:        images.GetSha(images.GetFlowCollectorImageName()),
+				Repository: "https://github.com/skupperproject/skupper",
+			},
+			{
+				Name: images.GetPrometheusServerImageName(),
+				SHA:  images.GetSha(images.GetPrometheusServerImageName()),
+			},
```

Comment:
I guess we also need the site-controller image here.

Reply:
updated in f562f47932c2321c75e9dc0a9bc05d00f2cc4d28